.circleci/config.yml

```diff
@@ -12,8 +12,50 @@ jobs:
 
     - run: ./architect build
 
+    - persist_to_workspace:
+        root: .
+        paths:
+        - ./e2e-harness
```

Comment:
How e2e-harness is available in the CI? I don't see any download step. I think I'd be nice to run it as docker image.

Reply:
It is build by architect in the first step. when used in another repo it will be downloaded, follows the same pattern as archiyect does. IMO using an image for this is a little bit cumbersomr, the client repos would need to know a lot of details about the inner workings, which directories to mount, which env vars to set, etc. The binary encapsulates these details, WDYT?